src/constants/util.rs

```diff
@@ -37,7 +37,7 @@ pub fn evaluate<C: CurveAffine>(x: u8, coeffs: &[C::Base]) -> C::Base {
     coeffs
         .iter()
         .rev()
-        .fold(C::Base::default(), |acc, coeff| acc * x + coeff)
+        .fold(C::Base::zero(), |acc, coeff| acc * x + coeff)
```

Comment:
We can also use `reduce` here (which initializes `acc` with the first element of the iterator) to avoid a multiplication:
```suggestion
        .reduce(|acc, coeff| acc * x + coeff)
        .unwrap_or_else(C::Base::zero)
```